example/src/shared/utils.ts

```diff
@@ -10,3 +10,12 @@ export const getShortUsername = (username: string) => {
     .join('')
     .toUpperCase();
 };
+
+/**
+ * Used to check whether string contains only whitespaces.
```

Comment:
No need to document that, it's in the function name and the types.
Comments like this make code less readable. In time the function might change and we'll most likely forget to update the comment.

Good comments are explaining things that are not obvious (for example "I'm doing this like that because there is an issue #1234 with RN"). What the function does should be obvious from the function name and you're doing a good job creating small, well named functions 👍 

Also I'm adding comments like this one but to the public API to automatically generate public docs. But this function not a public API.

Reply:
Remved.